index.js

```diff
@@ -65,14 +65,14 @@ module.exports = function CaptainsLog(overrides) {
     // If no reasonable alternative is possible, don't log
     var nullLog = function() {};
 
-    logger.debug = logger.debug || nullLog;
-    logger.info = logger.info || nullLog;
-    logger.warn = logger.warn || logger.error || nullLog;
-    logger.error = logger.error || nullLog;
-    logger.crit = logger.crit || logger.error || nullLog;
-    logger.verbose = logger.verbose || nullLog;
-    logger.silly = logger.silly || nullLog;
-    logger.blank = logger.blank || nullLog;
+    logger.debug = logger.debug ? logger.debug.bind(logger) : nullLog;
+    logger.info = logger.info ? logger.info.bind(logger) : nullLog;
+    logger.error = logger.error ? logger.error.bind(logger) : nullLog;
+    logger.warn = logger.warn ? logger.warn.bind(logger) : logger.error;
```

Comment:
looks like we kept the "error" fallback, but lost the nullLog short-circuit here

Reply:
Also missing a `.bind` right?  so like `logger.warn? logger.warn.bind(logger) : logger.error? logger.error.bind(logger) : nullLog` ?

Reply:
In the `warn` method? well at the time `warn` is being set, `error` should be whether the `error` function if it exists, or the `nullLog` assigned in the `error` shortcut, am I correct?

Reply:
@mikermcneil In case you didn't see my comments up there

Reply:
@mikermcneil I see what you say, I've added the bindings